check.go

```diff
@@ -109,25 +108,30 @@ func (c *C) stopNow() {
 // logger is a concurrency safe byte.Buffer
 type logger struct {
 	sync.Mutex
-	writer bytes.Buffer
+	buffer bytes.Buffer
+	output io.Writer
+	verbosity uint8
 }
 
 func (l *logger) Write(buf []byte) (int, error) {
 	l.Lock()
 	defer l.Unlock()
-	return l.writer.Write(buf)
+	if l.verbosity > 1 {
+		l.output.Write(buf)
```

Comment:
That was also ignored in the original. It just used to happen in the reporter, now it's moved to the logger.
I'm not sure how to handle it, because maybe the logger cares only about the buffer, and the output is something for a best-effort. But anyway we'll get to that once we get to the logger.